device/globals/inventory.gd

```diff
@@ -22,19 +20,26 @@ func open():
 		return
 	sort_items()
 	show()
-	get_node("animation").play("show")
+	if has_node("animation"):
+		get_node("animation").play("show")
 
 
 func close():
 	if !is_visible():
 		return
-	if get_node("animation").is_playing():
-		return
-	#printt("closing inventory")
-	print_stack()
-	get_node("animation").play("hide")
-	get_node("look").set_pressed(false)
+
+	var animation = get_node("animation")
+	if animation:
+		if animation.is_playing():
+			return
+		animation.play("hide")
+
+	# XXX: What is this `look` node? A verb menu thing?
+	if has_node("look"):
+		get_node("look").set_pressed(false)
```

Comment:
I'm guessing that if the inventory actions thing was working, it would have actions defined by adding buttons with their named actions under the actions node, similar to the action menu, and look would be the pre-selected action.